superset-frontend/src/explore/components/controls/DndColumnSelectControl/ColumnSelectPopover.tsx

```diff
@@ -149,54 +138,42 @@ const ColumnSelectPopover = ({
         <Tabs.TabPane key="saved" tab={t('Saved')}>
           <FormItem label={t('Saved expressions')}>
             <StyledSelect
+              ariaLabel={t('Saved expressions')}
```

Comment:
nit: to avoid either one of these being updated out of sync with the other, could we introduce a `const HEADER_TEXT = t('Saved expressions')` or similar and then reference that in both the `FormItem` and `ariaLabel`?